new_arrivals_chi/app/templates/info.html

```diff
@@ -20,15 +20,15 @@
 {% block content %}
 
 <form action="/info" method="GET">
-    <button type="submit" class="language-button">{{ 'Español' if language == 'en' else 'English' }}</button>
+    <button type="submit" class="language-button">{{ 'Español' if language == 'es' else 'English' }}</button>
```

Comment:
A simpler way of doing this going forward may be to send all of the text for a given file in some object with the HTTP response. That way you can just use the language selected in in the app overall, and then not have to worry about it in the templates.

Reply:
@michplunkett I am currently working on substituting this with some translation JSONs in this branch:

https://github.com/uchicago-capp-30320/new-arrivals-chi/tree/add-json-languages

Does that make sense to you?